collect_app/src/main/java/org/odk/collect/android/activities/CrashHandlerActivity.kt

```diff
@@ -2,7 +2,7 @@ package org.odk.collect.android.activities
 
 import android.os.Bundle
 import androidx.activity.OnBackPressedCallback
-import org.odk.collect.crash_handler.CrashHandler
+import org.odk.collect.crashhandler.CrashHandler
```

Comment:
CheckStyle seemed to not like `selfie_camera` in Java code, so I changed this while I had that front of mind.